addons/repair/report/repair_templates_repair_order.xml

```diff
@@ -52,6 +52,7 @@
                                 <td>
                                     <p t-if="line.repair_line_type == 'add'"><i>(Add)</i> <span t-field="line.product_id">Product A</span></p>
                                     <p t-if="line.repair_line_type == 'remove'">(<i>Remove</i>) <span t-field="line.product_id">Product B</span></p>
+                                    <p t-if="line.repair_line_type == 'recycle'">(<i>Recycle</i>) <span t-field="line.product_id">Product C</span></p>
```

Comment:
We have a newer `data-oe-demo` attribute to avoid cluttering the report with demo placeholders + so they don't get exported to be translated. Can you please adapt your change to use it?

```suggestion
                                    <p t-if="line.repair_line_type == 'recycle'">(<i>Recycle</i>) <span t-field="line.product_id" data-oe-demo="Product C"/></p>
```